components/formatter/InternalExternalIP.vue

```diff
@@ -0,0 +1,20 @@
+<script>
+import CopyToClipboardText from '@/components/CopyToClipboardText';
+export default {
+  components: { CopyToClipboardText },
+  props:      {
+    row: {
+      type:     Object,
+      required: true
+    },
+  },
+};
+</script>
+
+<template>
+  <span>
+    <CopyToClipboardText v-if="row.internalIp && row.internalIp.includes('.')" :text="row.internalIp" /><span v-else>None</span>
+    <span>/</span>
+    <CopyToClipboardText v-if="row.externalIp && row.externalIp.includes('.')" :text="row.externalIp" /><span v-else>None</span>
```

Comment:
Why the check for dot?  Won't work on an ipv6 address.

Reply:
Switched to v4/v6 checks using the existing ip dependency.